common/kvstore/src/test/java/org/apache/spark/util/kvstore/KVStoreIteratorSuite.java

```diff
@@ -36,9 +36,9 @@
 import org.slf4j.LoggerFactory;
 import static org.junit.Assert.*;
 
-public abstract class DBIteratorSuite {
+public abstract class KVStoreIteratorSuite {
```

Comment:
I've just renamed this as it made me confused - I imagined DB as LevelDB but there's separate suite for LevelDB. KVStore sounds better to me.